time_series_arima.Rmd

```diff
@@ -0,0 +1,133 @@
+# Time Series Modeling with ARIMA in R
+
+William Yu
+```{r setup, include=FALSE}
+ knitr::opts_chunk$set(warning = FALSE, message = FALSE)
```

Comment:
Hey, please remove this.

Reply:
Hi, is there any way I can edit this without commiting pushing and merging again? Can I edit the file directly on here?

Reply:
Oh nevermind, I think I got it.